examples/terraform/nutanix/main.tf

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2022 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+provider "nutanix" {
+  insecure = true
```

Comment:
Can we maybe make this configurable and default to `false`? There's hopefully Nutanix setups out there with valid TLS certs.

Reply:
Added `allow_insecure` field which is `false` by default.